values.yaml

```diff
@@ -2617,9 +2617,9 @@ commonAnnotations: { }
 
 ## @extra commonLabels Labels to add to all deployed objects
 ##
-commonLabels:
-  ## @param commonLabels.app.kubernetes.io/version Labels Deploy server version
-  app.kubernetes.io/version: "{{ .Chart.AppVersion }}"
+commonLabels: { }
+## @param commonLabels.app.kubernetes.io/version Labels Deploy server version
+## Eg. app.kubernetes.io/version: "{{ .Chart.AppVersion }}"
```

Comment:
Don't change the default values, we need them as they are currently

Reply:
Removed them, as they are rendered twice in the labels section in the statefulsets. Not needed to be given explicitly here. Its rendered once by the "common.labels.standard" and once by ".Values.commonLabels" conditional part.